.github/workflows/tests.yaml

```
@@ -46,70 +59,93 @@ jobs:
           make test-pylint
       - name: Run pyright
         run: |
-          sudo snap install --classic node
-          sudo snap install --classic pyright
           make test-pyright
-      - name: Install LXD
+
+  unit-tests:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.8", "3.9", "3.10"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python versions on ${{ matrix.platform }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Configure environment
+        run: |
+          echo "::group::pip install"
+          python -m pip install -U .[dev]
+          python -m pip install -U -e .
```

```suggestion
          python -m pip install -U -e .[dev]
```

will do the same thing, right?

---

Yes, fixed!